lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -663,7 +663,8 @@ DefConstructor('\@@numbered@section{} Undigested OptionalUndigested Undigested',
         Warn("malformed", "ltx:$type", "Tried to open an unknown tag ltx:$type for numbered section"); } }
     $document->openElement($tagname,
       'xml:id' => CleanID($id),
-      inlist   => ToString($inlist));
+      inlist   => ToString($inlist),
+      (LookupValue('INDENT_FIRST') ? (class => 'ltx_nofirstchild') : ()));
```

Comment:
I am rather confused by the "nofirstchild" name. Is there a better one?

Reply:
For example, `ltx_indent_all_p` would be clearer (though I'm not sure that's the right intention - basing that on the new CSS rule).

Reply:
Probably is a better name, but the way it ends up working is to override the firstchild behavior in css. Which of course has the effect (currently) of not indenting.

Reply:
Actually worse: the use in `pruneEmpty` (which I wrote first) really means there is no first child, since it got removed; so what looks like the first child is actually the second. The use here does mean indent the first child.  I suppose it would be clearer intention to introduce another class name, even though there will be extra css rules.

Reply:
From the perspective of someone reading classes to style with CSS, names that talk about `ltx_indent_first_p` and `ltx_noindent_first_p` are both concise and clear, so I think that is a very healthy renaming direction.

Encountering `ltx_nofirstchild` is quite confusing (to me) without a lot of extra context. If the emphasis is on the first paragraph getting pruned, maybe `ltx_pruned_first_p` is a first name idea?

Reply:
Sure, it is "needlessly specific" if you can resolve the ambiguity that the first means the `p` and not the local title (e.g. section heading).

Good enough to merge in either case.